written_responses.md

```diff
@@ -1,5 +1,11 @@
 1. What is a regular expression and how does it work?
 
```

Comment:
Well written answers Bryce. I like how you start at a high-level then get into details and more technical aspects of each structure as you go along 👍